lib/utils.ts

```diff
@@ -4,3 +4,22 @@ import { twMerge } from "tailwind-merge";
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs));
 }
+
+export function getAvatarInitials(fullName: string) {
+  const names = fullName.trim().split(/\s+/);
+  let initials = "";
+
+  if (names.length === 1) {
+    // If there's only one name, use the first and last character of this name
+    initials =
+      names[0][0] +
+      (names[0].length > 1 ? names[0][names[0].length - 1] : names[0][0]);
```

Comment:
I think it makes more sense to use the first two characters ratehr than the last character
and below, use the first character of the last name too

you can use .at(-1) to get the last element, it makes it much easier to read